src/steps/os/archlinux.rs

```diff
@@ -161,15 +195,14 @@ pub fn get_arch_package_manager(ctx: &ExecutionContext) -> Option<Box<dyn ArchPa
         config::ArchPackageManager::Autodetect => YayParu::get("paru", &pacman)
             .map(box_package_manager)
             .or_else(|| YayParu::get("yay", &pacman).map(box_package_manager))
-            .or_else(|| {
-                Trizen::get()
-                    .map(box_package_manager)
-                    .or_else(|| Pacman::get(ctx).map(box_package_manager))
-            }),
+            .or_else(|| Trizen::get().map(box_package_manager))
```

Comment:
This doesn't pass `cargo fmt`

Reply:
I did forget to add changes from `cargo fmt`, but this wasn't one of them :thinking: 
Currently passes (rustfmt 1.4.38-nightly)